neo/NeoSystem.cs

```diff
@@ -74,17 +74,10 @@ public void StartConsensus(Wallet wallet, Store consensus_store = null, bool ign
             Consensus.Tell(new ConsensusService.Start { IgnoreRecoveryLogs = ignoreRecoveryLogs }, Blockchain);
         }
 
-        public void StartNode(int port = 0, int wsPort = 0, int minDesiredConnections = Peer.DefaultMinDesiredConnections,
-            int maxConnections = Peer.DefaultMaxConnections, int maxConnectionsPerAddress = 3)
+        public void StartNode(NodeStartConfig cfg)
         {
```

Comment:
I changed this to a class because is easy to extend in the future without problems, and in fact, we used a class `Peer.Start` before